lib/strings.md

```diff
@@ -5,20 +5,40 @@ instructor_notes: Feel free to re-organize the headings (or add/remove headings)
 
 # What is a String?
 
-Replace me with your response...
+A string, simply put, is a collection of special characters, letters, and/or numbers. Strings can be as a simple as a name to something longer like an entire sentence or paragraph.
```

Comment:
Within Ruby Strings, letters, numbers, and special characters are all technically thought of as "characters".